### PR TITLE
[README] Add Japanese-translated version

### PR DESCRIPTION
**Pull request by MaySoMusician**
~*Note: the PR is still in progress, not ready for merge*~ Now I just marked this ready for review

This PR adds new README translated to Japanese as `README.ja.md`. This doesn't make any changes to type definitions.

### CODE REVIEW

**Review by amcasey — APPROVED** [review]
Thanks, @MaySoMusician!  This was clearly a lot of work.  Since I, unfortunately, do not speak Japanese I just samples bits of the doc with the aid of machine translation.  On that basis, it LGTM.

**Comment by MaySoMusician** [comment]
@amcasey Thanks for your approval. I hope it'll get merged!

**Review by sasurau4 — COMMENTED** [review]
いくつか指摘しました :+1:

File: README.ja.md [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+| 3.3        | 2019年1月  | 2021年1月    |
+| 3.4        | 2019年3月  | 2021年3月    |
+| 3.5        | 2019年5月  | 2021年5月    |
+| 3.6        | 2019年8月  | 2021年8月    |
+| 3.7        | 2019年11月 | 2021年11月   |
+| 3.8        | 2020年2月  | 2022年2月    |
+| 3.9        | 2020年5月  | 2022年5月    |
+| 4.0        | 2020年8月  | 2022年8月    |
+
+`@types` パッケージには、サポートする TypeScript のバージョンを明示的に指定するタグがあるため、多くの場合はサポート期間内のバージョン用のパッケージを入手できます。
```
[the remark this answers is not in the document]
Comment (MaySoMusician) [reply README.ja.md]: ここは、predateの意味する「前」が逆でしたね（サポート終了「前」ではなく、サポート期間より「前」にリリースされたバージョン）。ご指摘ありがとうございます

**Comment by MaySoMusician** [comment]
@sasurau4 ありがとうございます！ より自然な言い回しだったので apply suggestions しました

**Review by sasurau4 — APPROVED** [review]
LGTM :+1:

**Review by peterblazejewicz — APPROVED** [review]

**Comment by orta** [comment]
OK cool, I'm merging - I trust @sasurau4 from their TS website work

**Pull request merged**